content/source/docs/cloud/api/workspace-resources.html.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "cloud"
+page_title: "Workspaces - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+[JSON API document]: /docs/cloud/api/index.html#json-api-documents
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# Workspace Resources API
+
+## List Workspace Resources
+
+`GET /workspaces/:workspace_id/resources`
+
+Parameter            | Description
+---------------------|------------
+`:workspace_id`      | The ID of the workspace to retrieve resources from. Obtain this from the [workspace settings](../workspaces/settings.html) or the [Show Workspace](./workspaces.html#show-workspace) endpoint.
```

Comment:
```suggestion
`:workspace_id`      | The ID of the workspace to retrieve resources from. Obtain this from the [workspace settings](../workspaces/settings.html) or the [show workspace](./workspaces.html#show-workspace) endpoint.
```

I'm unsure of preference here, so will defer to @laurapacilio, but with `workspace settings` lowercased, do we want the same with the other link?

Reply:
I agree with lower case :)  I think we don't have a reason to capitalize this here.

Reply:
Oh I didn't realize I made this different here.  Thank you!  I would have actually argued the opposite (capitalize _Workspace Settings_ due to the proper naming with the _the_ but I think as long as we're consistent it's good 👍